spyder/config/main.py

```diff
@@ -83,6 +83,8 @@
               'memory_usage/timeout': 2000,
               'cpu_usage/enable': False,
               'cpu_usage/timeout': 2000,
+              'clock/enable': True,

```

Comment:
I think this should be `False` by default because it's something useful only when working in fullscreen mode, as you pointed out.

Reply:
Agreed!

Reply:
I agree!